libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -617,6 +617,11 @@ userdata AP_Camera::camera_state_t field zoom_type uint8_t'skip_check read
 userdata AP_Camera::camera_state_t field zoom_value float'skip_check read
 userdata AP_Camera::camera_state_t field focus_type uint8_t'skip_check read
 userdata AP_Camera::camera_state_t field focus_value float'skip_check read
+userdata AP_Camera::camera_state_t field tracking_type uint8_t'skip_check read
```

Comment:
let's try and make this a vector2f data

Reply:
camera bindings have been changed to use Vector2f